aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -741,7 +741,7 @@ module aptos_framework::account {
         );
         table::add(address_map, new_auth_key, originating_addr);
 
-        if (std::features::module_event_migration_enabled()) {
```

Comment:
why we change this?

Reply:
The same reason as: https://github.com/aptos-labs/aptos-core/pull/14615/files#r1841101915

Reply:
@lightmark , I reverted this change. I just removed the new feature flag as we don't need to separate account & coin events from token events. We now use just one flag: `MODULE_EVENT_MIGRATION`.